### PR TITLE
Add complete translation for PT-BR.

### PR DESCRIPTION
**Pull request by yds12**
Hello, I am a native Brazilian Portuguese speaker and I translated your PT-BR language file. Does this project accept PRs?
Let me know if there is anything I can fix so that this PR can be merged.

### CODE REVIEW

**Comment by Flaburgan** [comment]
Hello @yds12 and thank you for contributing to Axolotl!
Yes of course this project accept pull requests. Did you take time to install locally the environment and try your work?

File: po/pt-rBR.po [hunk 1]
```diff
@@ -13,25 +13,25 @@ msgstr ""
...
-msgstr ""
+msgstr "Sobre Axolotl"
 
 #: src/components/AddContactModal.vue:5 src/components/AddContactModal.vue:22
 msgid "Add"
-msgstr ""
+msgstr "Adicionar"
 
 #: src/components/AddGroupMembersModal.vue:51
 #, fuzzy
```
Comment (nanu-c) [inline po/pt-rBR.po:27]: Thanks for your translations, it's really appreciated. In order to pick up the strings, the `#, fuzzy` needs to be removed, too, can you to this before we merge it?

**Comment by yds12** [comment]
Hello @Flaburgan, I just tried to run locally but the app shows me a 404, here is a screenshot: https://imgur.com/a/Cif7cq7.

Hi @nanu-c do you mean to remove all the lines containing `fuzzy`, or just the ones where I added something?

**Comment by nanu-c** [comment]
> Hello @Flaburgan, I just tried to run locally but the app shows me a 404, here is a screenshot: https://imgur.com/a/Cif7cq7.
In order to test the translations it's best to run in the `axolotl-web` folder `npm run translate` and then `npm run serve` to see the changes when you point your browser to http://localhost:8080

> Hi @nanu-c do you mean to remove all the lines containing `fuzzy`, or just the ones where I added something?
We should remove all the fuzzy lines, whenever the translation is correct. So everywhere you changed something + in the places where the translations are already good.

**Comment by yds12** [comment]
I just pushed the changes removing the `fuzzy` marks.
@nanu-c I did that, then on the browser I clicked on a button that opened the desktop (Electron?) app.

**Comment by Flaburgan** [comment]
I just pulled your changes and compiled them, but the translations aren't displayed. Why is the file named pt-**r**BR? What's that r, isn't that a typo @nanu-c ?

**Comment by yds12** [comment]
@Flaburgan I also suspected it was a typo, but I am not sure. Brazilian Portuguese is usually pt-BR. Maybe the place that pulls the translation file doesn't have the typo?

**Comment by Flaburgan** [comment]
I fixed the local and merged. Thank you! :cookie:

**Pull request merged**